keithley2600/keithley_driver.py

```diff
@@ -763,7 +763,7 @@ def _convert_input(self, value: Any) -> str:
             # convert some iterables to a TSP type list '{1,2,3,4}'
             return "{" + ", ".join([str(v) for v in value]) + "}"
         elif isinstance(value, (int, float, np.number)) and not isinstance(
-            value, np.complex
+            value, complex
```

Comment:
I'm not quite sure why the `not isinstance(value, complex)` check was introduced in the first place. Its probably safe to remove it all together, if `isinstance(value, (int, float, np.number))` is true, `isinstance(value, complex)` will always be false anyways.

Reply:
Yeah just checked, a complex number is never an instance of int, float or np.number. Will take out that check :)